float64[] cartesian_pose
duration time_from_start
